### PR TITLE
A023811

### PR DESCRIPTION
**Pull request by BEYMOHAMED**

### CODE REVIEW

File: oeis.py [hunk 1]
```diff
@@ -169,6 +169,19 @@ def phi(n):
     return [phi(x) for x in range(start, start + limit)]
 
 
+@oeis
+def A023811(start,limit):
```
Comment (JulienPalard) [inline oeis.py]: Missing: Function docstring (see other functions). Test "oeis.py --list" to see why it's usefull.

File: oeis.py [hunk 2]
```diff
@@ -169,6 +169,20 @@ def phi(n):
     return [phi(x) for x in range(start, start + limit)]
 
 
+@oeis
+def A023811(start, limit):
+    def largest_metadrome(n):
```
Comment (JulienPalard) [inline oeis.py]: Missing docstring, see `oeis.py --list` to see why it's usefull.

**Comment by JulienPalard** [comment]
Hi @BEYMOHAMED, can you please resolve conflicts on this PR?

File: oeis.py [hunk 3]
```diff
@@ -170,6 +170,23 @@ def A000040(start: int = 0, limit: int = 20) -> Collection[int]:
 
 
 @oeis
+def A023811(start, limit):
```
Comment (JulienPalard) [inline oeis.py]: According to the travis report, you should add type annotation here, as all other functions did.

File: oeis.py [hunk 4]
```diff
@@ -170,6 +170,23 @@ def A000040(start: int = 0, limit: int = 20) -> Collection[int]:
...
+        return result
+
+    tab = []
+    for n in range(start, start + limit):
+        tab.append(largest_metadrome(n))
+    return tab
+
+
+@oeis
+
```
Comment (JulienPalard) [inline oeis.py]: A blank line is not pretty here, and `black` from `Travis` is asking for you to remove it.

**Comment by JulienPalard** [comment]
ping.

**Comment by BEYMOHAMED** [comment]
hello Julien,
I'm having some troubles while trying to push my sequence, I did the merge
with the master branch, I have no conflict but when I do the push commande
I have a error "hint: Updates were rejected because the tip of your current
branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again."

Le mer. 4 déc. 2019 à 12:38, Julien Palard <notifications@github.com> a
écrit :

**Comment by JulienPalard** [comment]
So maybe try the hint: `git pull origin:A023811`? Without knowing your full setup it's complicated, if it does not work please provide:
```
git branch -vvv
git remote -v
```

**Comment by BEYMOHAMED** [comment]
git branch -vvv :
* A023811 286abce Merge branch 'master' of
https://github.com/JulienPalard/oeis into A023811
  master  ca3d5ec [origin/master] delete A008592 duplicate

git remote -v :

origin  https://github.com/BEYMOHAMED/oeis.git (fetch)

origin  https://github.com/BEYMOHAMED/oeis.git (push)

upstream        https://github.com/JulienPalard/oeis.git (fetch)
upstream        https://github.com/JulienPalard/oeis.git (push)

**Comment by JulienPalard** [comment]
You mean `git pull origin:A023811` did not worked? What did it said?

File: oeis.py [hunk 5]
```diff
@@ -169,6 +169,22 @@ def A000040(start: int = 0, limit: int = 20) -> Collection[int]:
     return result
 
 
+@oeis
+def A023811(start, limit):
```
Comment (JulienPalard) [inline oeis.py]: `start` and `limit` should have a default value, and like any other implementations in this file, type annotations, like:
```suggestion
def A023811(start: int = 0, limit: int = 20) -> Collection[int]:
```

**Comment by BEYMOHAMED** [comment]
ssh: Could not resolve hostname origin: nodename nor servname provided, or
not known
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

**Comment by JulienPalard** [comment]
Please take a look at https://github.com/JulienPalard/oeis/pull/43/files or locally using `git diff upstream/master`: you commited the same function twice. first at line 157, next at line 188.

File: oeis.py [hunk 6]
```diff
@@ -153,6 +153,22 @@ def A000040(start: int = 1, limit: int = 20) -> Collection[int]:
     return list(sieve[start : start + limit])
 
 
+@oeis
+def A023811(start: int = 0, limit: int = 20) -> Collection[int]:
+    "Largest metadrome (number with digits in strict ascending order) in base n."
+
+    def largest_metadrome(n):
```
Comment (JulienPalard) [inline oeis.py]:
```suggestion
    def largest_metadrome(n: int) -> int:
```

**Comment by BEYMOHAMED** [comment]
thanks for all the help.

**Comment by JulienPalard** [comment]
:+1:

**Pull request merged**